src/components/Vega.svelte

```diff
@@ -209,6 +237,12 @@
     }
   });
 
+  afterUpdate(() => {
+    if (!patchSpec) {
```

Comment:
is there a difference to triggering `vega.view.resize().runAsync()`? cause in this in case of the small multiples N events are triggered